Jenkinsfile.rh.release

```diff
@@ -24,9 +24,51 @@ properties([
   ]),
 ])
 
+String version = ''
+String checksum = ''
+
+void updateIQServerVersionAndChecksum(String version, String checksum) {
+  def dockerFile = readFile(file: 'Dockerfile.rh')
+  def metaShortVersionRegex = /(release=")(\d\.\d{1,3}\.\d)(" \\)/
+  def versionRegex = /(ARG IQ_SERVER_VERSION=)(\d\.\d{1,3}\.\d\-\d{2})/
+  def shaRegex = /(ARG IQ_SERVER_SHA256=)([A-Fa-f0-9]{64})/
+  dockerFile = dockerFile.replaceAll(metaShortVersionRegex,
+      "\$1${version.substring(0, version.indexOf('-'))}\$3")
+  dockerFile = dockerFile.replaceAll(versionRegex, "\$1${version}")
+  dockerFile = dockerFile.replaceAll(shaRegex, "\$1${checksum}")
+
+  writeFile(file: 'Dockerfile.rh', text: dockerFile)
+}
+
+void commitAndPushChanges(String version) {
```

Comment:
appears if you stop modifying Dockerfile.rh this also becomes unnecessary as part of releasing the image

Reply:
Commit: a2a41fa10702ac5eaf527a8818f39a19baf2e028